features/support/after-hook.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const sdapi = require('./sdapi');
+
+/**
+ * After hooks
+ * @return
+ */
+function afterHooks() {
+    this.After('@apitoken', () => {
```

Comment:
The one that cares about a potential conflict also calls the cleanup in `before`, my understanding is that this is just to leave the server in a cleaner state provided everything goes well.